mentors/csharp/v.json

```diff
@@ -7,4 +7,14 @@
     "avatar_url": null,
     "bio": "I've been using C# professionally for 8 years and love its versatility!"
   }
+],
```

Comment:
The comma at the end of this line should be at the end of the previous line and this line should be deleted.

```suggestion
```